install/kubernetes/istio-auth.yaml

```diff
@@ -673,6 +673,8 @@ spec:
       valueType: STRING
     request.auth.presenter:
       valueType: STRING
+    request.api_key:
+      valueType: STRING
```

Comment:
Do you need change the template instead?

https://github.com/istio/istio/tree/master/install/kubernetes/templates

Reply:
This is auto generated by install/updateVersion.sh